### DISCUSSION TITLE
Config freezes when selecting Play

### DISCUSSION THREAD

**Discussion by PerrigoGames** [wcko87/beatoraja-english-guide#44] [beatoraja questions / troubleshooting]
I'm having trouble getting the actual game part to launch from the config menu.  I'm launching the config bat from Powershell, the JavaFX part seems to work alright, but when I click the Play button, the JavaFX window becomes unresponsive.  There is no further output to my shell when the button is clicked.

OS:  Windows 10 build 19045
Java:  openjdk 21.0.2 2024-01-16 LTS
OpenJDK Runtime Environment (build 21.0.2+14-LTS)
OpenJDK 64-Bit Server VM (build 21.0.2+14-LTS, mixed mode, sharing)

UTF-8 bat is being used
The player name has been set
Songs have been added in Resource and the BMS database has been updated

**Comment by wcko87** [comment]
Apologies for the slow response. Since the issue is occurring when you press Play, it might be a skin-related issue. Have you tried running it with the default skin?

**Reply by PerrigoGames** [reply]
And I apologize for the slow follow-up.  It turns out the Illegal song prompt was opening UNDER the config window, and it required alt-tabbing to see it.
